rllib/algorithms/dqn/dqn.py

```diff
@@ -307,13 +341,59 @@ def validate(self) -> None:
                     " used at the same time!"
                 )
 
+        # Validate that we use the corresponding `EpisodeReplayBuffer` when using
```

Comment:
Can we add a TODO here (and also in SAC in the respective line) that we need to implement a MultiAgentEpisodeReplayBuffer to enable SAC/DQN for multi-agent cases?

Reply:
Let's do this in an extra PR. I already made an issue for this some time ago: https://github.com/ray-project/ray/issues/42872